packages/docs/pages/reference/elements/tabular.mdx

```diff
@@ -8,12 +8,179 @@ import {
 
 # `<tabular>`
 
-Short description of what the `<tabular>` tag does.
+The `<tabular>` tag is the workhorse tag for displaying a collection of cells
```

Comment:
It's worth adding that what most people call a "table", pretext calls a "tabular"